auth-web/src/stores/business.ts

```diff
@@ -90,10 +88,11 @@ export const useBusinessStore = defineStore('business', () => {
       ...addConditionalProperty(resp.modified, { modified: resp.modified }),
       ...addConditionalProperty(resp.modifiedBy, { modifiedBy: resp.modifiedBy }),
       ...addConditionalProperty(resp.nrNumber, { nrNumber: resp.nrNumber }),
+      ...addConditionalProperty(resp.effectiveDate, { effectiveDate: resp.effectiveDate }),
       ...addBooleanProperty(resp.adminFreeze, false, 'adminFreeze'),
       ...addBooleanProperty(resp.goodStanding, true, 'goodStanding'),
       ...addBooleanProperty(resp.inDissolution, false, 'inDissolution'),
-      ...addConditionalProperty(resp.state, { status: resp.state })
+      status: resp.state || resp.draftStatus
```

Comment:
I updated this:
if there is a state ("ACTIVE" or "HISTORICAL"), business.status = state
if there is a draftStatus, the business.status would be the draftStatus.
It should work, but after I update it, there is an error:
![image](https://github.com/user-attachments/assets/6c929e0d-09fb-42f7-85f3-c83a8a551c3b)
Could you please help debug? @severinbeauvais  Thanks!

Reply:
You have a null string on which `toUpperCase` is being called. Check your data. Use conditional chaining if needed.

Reply:
Even when I set the status to be null, or empty string, there is no error. The error only shows when I set the status to be resp.draftStatus.